src/host/index.ts

```diff
@@ -33,6 +33,7 @@ export class Host {
 
   // Route incoming request to a plugin
   async handlePlugin(method: string, args: any[]) {
+    if (method.startsWith('nvim_buf_')) return
```

Comment:
Just for reference, our plan is to reserve the `nvim_` prefix for events. Also, all events will have the `_event` suffix.

So maybe it makes sense to ignore `nvim_`, not just `nvim_buf_` ?

Reply:
Sure.